composer.json

```
@@ -21,14 +21,14 @@
     "homepage": "https://docs.sonata-project.org/projects/form-extensions",
     "require": {
         "php": "^8.0",
-        "symfony/event-dispatcher": "^5.4 || ^6.2",
-        "symfony/form": "^5.4 || ^6.2",
-        "symfony/options-resolver": "^5.4 || ^6.2",
-        "symfony/property-access": "^5.4 || ^6.2",
-        "symfony/security-csrf": "^5.4 || ^6.2",
-        "symfony/translation": "^5.4 || ^6.2",
+        "symfony/event-dispatcher": "^5.4 || ^6.2 || 7.0.*",
```

```suggestion
        "symfony/event-dispatcher": "^5.4 || ^6.2 || ^7.0",
```
To allow futur Symfony minor version.